pom.xml

```diff
@@ -96,6 +96,7 @@
         <module>core/trino-spi</module>
         <module>docs</module>
         <module>lib/trino-array</module>
+        <module>lib/trino-errorprone</module>
```

Comment:
I don't think it should be a Trino module.

It's better suited as a separate lib.

Reply:
I would start as a module, to iterate faster (like we did with extensions for testng). Then once we see a need we could extract it to separate project.